soc/st/stm32/stm32wb0x/Kconfig.defconfig

```diff
@@ -8,4 +8,13 @@ if SOC_SERIES_STM32WB0X
 config NUM_IRQS
 	default 32
 
+config BT_AUTO_PHY_UPDATE
+	default n
+
+config BT_AUTO_DATA_LEN_UPDATE
+	default n
```

Comment:
Seems like you opted to just do this, and trust that no app will try to re-enable them? I.e. you don't even want build asserts?

Reply:
Yes. We decided to allow these options to be set since the error is not critical and we plan to fix it in a future PR, after discussion with the stack team.